docs/05-PipelineLaunchOptions.md

```diff
@@ -0,0 +1,99 @@
+# 5.0 - Pipeline Launch options
+
+Scans upload into LORIS and insertion pipeline's triggering can be done in a few 
+ways depending on each project's needs. The best choice will depend on whether 
+for example data is collected retrospectively or prospectively, and/or whether 
```

Comment:
I would remove the "for example". Makes the sentence more complicated to read.

Or could rephrase like this?
"The best choice will depend on the study workflow and protocol. For example, it will depend on whether data is collected retrospectively or prospectively, ..."